be/src/common/config.h

```diff
@@ -821,6 +821,9 @@ CONF_mBool(orc_coalesce_read_enable, "true");
 // Default is 8MB for tiny stripe threshold size
 CONF_Int32(orc_tiny_stripe_threshold_size, "8388608");
 
+// orc load parameters
```

Comment:
```suggestion
// When ORC file size is smaller than orc_loading_buffer_size, we'll read the whole file at once instead of reading a footer first.
```

Reply:
done